Source/WebInspectorUI/UserInterface/Views/MultiSidebar.js

```diff
@@ -322,7 +364,48 @@ WI.MultiSidebar = class MultiSidebar extends WI.Sidebar
     {
         this.dispatchEventToListeners(WI.Sidebar.Event.WidthDidChange, {sidebar: event.target, newWidth: event.data.newWidth});
     }
+    
+    _recalculateHeight(newHeight)
+    {
+        console.assert(newHeight);
+        let maximumHeight = this.element.parentElement.offsetHeight - WI.TabBrowser.MinimumHeight;
```

Comment:
There's no need to use `WI.TabBrowser.MinimumHeight` here because you set `#tab-browser {max-height: 110px}` in `Main.css`

The layout engine will apply that constraint when the sidebar height is too large. 


```diff
- let maximumHeight = this.element.parentElement.offsetHeight - WI.TabBrowser.MinimumHeight;
+ let maximumHeight = this.element.parentElement.offsetHeight;
```

This means you can remove the comment from `Main.css` and  `WI.TabBrowser.MinimumHeight = 110;` from `TabContentView`

Reply:
Did the latter.